### PR TITLE
Fix FIelds on Appointment Request

### LINKED ISSUE #140: Fix fields on appointment request
#### Description
- anything in the database that isn't on the form that mentees fill out should be removed
- remove mentorship goals
- remove ethnicity
- add boolean allowTexts
- add boolean allowCalls

### PR DESCRIPTION
**Pull request by AzureQDragon**
## Status:
:rocket: Ready

## Description :star2:

Fixes #140

### CODE REVIEW

**Review by AngelaLuo49021 — APPROVED** [review]
Good stuff

File: backend/api/views/appointment.py [hunk 1]
```diff
@@ -26,37 +26,39 @@ def get_requests_by_mentor(mentor_id):
...
-        organization=data.get("organization"),
-    )
-
-    time_data = data.get("timeslot")
-    new_appointment.timeslot = Availability(
-        start_time=time_data.get("start_time"), end_time=time_data.get("end_time")
-    )
+    try:
+        data = request.get_json()
+        print(data.get("allow_calls"))
```
Comment (chyku) [inline backend/api/views/appointment.py:31]: delete print lol

File: backend/api/views/appointment.py [hunk 2]
```diff
@@ -26,37 +26,39 @@ def get_requests_by_mentor(mentor_id):
...
+        )
+
+        time_data = data.get("timeslot")
+        new_appointment.timeslot = Availability(
+            start_time=time_data.get("start_time"), end_time=time_data.get("end_time")
+        )
+
+        new_appointment.save()
+    except Exception as e:
+        print(e)
```
Comment (chyku) [inline backend/api/views/appointment.py:60]: is this needed here still? generally when theres an exception u wanna create a response with error code + message but idk if u left this while debugging something

**Review by chyku — CHANGES_REQUESTED** [review]
looks fine but some cleanup needed, also we're gonna need to add these fields to the database before this code is pushed to master

**Review by chyku — APPROVED** [review]

**Pull request merged**